.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: python
+python:
+- '3.6.3'
+before_script:
+- pip install pybuilder
+script: pyb package publish
+branches:
+  only:
+  - master
+before_deploy:
+  - cd target/dist/janusgraph_python
+deploy:
+  provider: pypi
+  user: debasishdebs
+  server: https://test.pypi.org/legacy/
+  skip_cleanup: true
+  true:
+    branch: master
+  password:
+    secure: f0TQzX+wDDOT3LV76MO3ba8xLBYbcrne509HpA3w6nQKCl4OZobjC0SXG39ATAp6dq1m098FEioVB4usw9NVcc+fpeZm0jmkKq9fUvh9j0TxDC/wT0L6Rn6Cn2BnL8i/xBWkn027a4/FHFSgEqQgOycYuTcOggfyCFIbi5FflBSScjkqY6jlwXHLmIBRHgvHulkXLOOVFS5jDlI9rHqo87Ejj/A47IRttdjoz3Vt4c7ECTn+KZGsVRhvwLAraT/pRn4Hq1IuG/Xiykd2GAbCRBPeTTRkL2NCBEgjWEJSXnBRoKe+fAdI1J/sxyYd8eBx2KQLtiUkP4yPJELaAe6UoypVUvTVV3B8M8vDG3/fJlxBA3MyuOI6C2F4ilnHSdzCG/IVMBEE5lB2Jy4b60dbOxoJxU1gerfGYIo9SPNV0Q6FtCpWzjUmVfNpeEEN9pi9pPqjcaOaauk5SlMTNu0RieO2oa9lF2sK9Bfh7Uk6+DJMEDELKZytlDMHNMFwvakDwEBDSDmOxnhxkvn6lbGWx/dNXwe07nXpl/FVSVsJaGvoh8mrFMkGFTcoqHrn2nTMLKoKlbMQeCWfhTld1OfUMsolYjdk9yuowsFN8z+weXb1FpsVeXppCUZZ34nHdtX49vTkXKx0e6tJNYojsPGu4aDtz50dsXI6qfUIo7j3CGM=
```

Comment:
@sjudeng : By stub do you mean dummy blank values? But then our repo won't be built using Travis CI.
Or do u mean hash those as well? I'm okay with hashing username, but server address (in this case its official PyPi repo), I don't see enough reason to hash that.

Reply:
Aren't these credentials just required for the deployment rather than build? Should deployment be limited just to tags and should we use group (janusgraph) rather than individual credentials for deployment? I'm not sure if we have group credentials yet but if not I'm sure it's something we could do in time for the first tagged release.

Reply:
Yes @sjudeng  those credentials are required for deployement. Well currently I use my personal account to deploy to Test PyPi.
If you think we can have a janusgraph group credentials ready for deployement, I can change to those credentials once provided, along with updating deployement server from test pypi to pypi (official)

Reply:
Thanks for the responses. @mbrukman made [a comment](https://github.com/JanusGraph/janusgraph-python/pull/4/files#r215827481) just to add a `TODO` here for updating the credentials, which makes sense to me too on this.